tests/tests.js

```diff
@@ -305,7 +306,17 @@ exports.defineAutoTests = function () {
             //         - 'httpssss://example.com'
             //         - 'apache.org', with subdomains="true"
             //         - 'cordova-filetransfer.jitsu.com'
-            describe('download', function() {
+            describe('download', function () {
+                var originalTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
+
+                beforeEach(function () {
+                    // Download tests need a larger time interval to run successfully
+                    jasmine.DEFAULT_TIMEOUT_INTERVAL = 10000;
```

Comment:
We've switched to moving Jasmine 2.2 which provides a nice syntax for modifying default timeout duration: http://jasmine.github.io/2.2/introduction.html#section-51